routes/web.php

```diff
@@ -25,11 +25,12 @@
 
 Route::feeds();
 
-Route::get('/', [IndexController::class, 'show'])->name('home')->middleware(['installation:complete', 'migration:complete']);
-Route::get('/gallery', [IndexController::class, 'gallery'])->name('gallery')->middleware(['installation:complete', 'migration:complete']);
+Route::get('/', [IndexController::class, 'show'])->name('home')->middleware(['migration:complete']);
+Route::get('/gallery', [IndexController::class, 'gallery'])->name('gallery')->middleware(['migration:complete'])
+		->name('login');
 Route::match(['get', 'post'], '/migrate', [Administration\UpdateController::class, 'migrate'])
-	->name('migrate')
-	->middleware(['installation:complete', 'migration:incomplete']);
+		->name('migrate')
+		->middleware(['migration:incomplete']);
```

Comment:
I am still unsure about the changes in the middleware checks.

 1. Have the been necessary for this PR?
 2. Does this PR also work without these changes?
 3. Do we have a full assessment and understanding of the impact of these changes (this one and the three changes below)?

I don't mind changing it, if we now for sure that we need these changes of what these changes do. Otherwise I would prefer to pull them out into an independent PR. This way we can revert the PR the independent PR individually if we have to without reverting everything.

Reply:
1. no
2. yes
3. Install just makes sure that `.env` etc are properly set if I remember correctly.

Reply:
Tested, I will need to investigate further as it seems that the `'installation:complete'` was ignored for some reason.

Reply:
Yes, I do so, too. Move it out into another PR.

Reply:
Isn't it already out of the PR? O.o

Reply:
Why have you requested another review? The changes to the routes are still in this PR and the last commit is still mine. Did you miss to push your changes?

Reply:
> Why have you requested another review?  

Well because you set it to "requested changes" change. :|

Updated as per your wishes If I understood what you want correctly with https://github.com/LycheeOrg/Lychee/pull/1403/commits/9dd463ce6d8bb3588afe50a21ed961d20157c69f.